ambry-store/src/main/java/com.github.ambry.store/IndexValue.java

```diff
@@ -239,13 +239,27 @@ short getAccountId() {
     return accountId;
   }
 
+  /**
+   * @return the accountId as a string in the format of Account[id]
+   */
+  String getAccountIdString() {
```

Comment:
this does not belong here. The short can be parsed into a string outside of here.

Reply:
okay, I thought we needed such method, just like `partition.toString()` and `partition.toPathString()`. Let me remove it from the file.